orchestrator/main_loops.go

```diff
@@ -222,12 +236,16 @@ func (p *peggyOrchestrator) EthSignerMainLoop(ctx context.Context) (err error) {
 func (p *peggyOrchestrator) BatchRequesterLoop(ctx context.Context) (err error) {
 	logger := p.logger.With().Str("loop", "BatchRequesterLoop").Logger()
 
-	// TODO: Change p.loopsDuration for something like 20 x average block time.
-	// We now send a batch request without checking for profitability, that'll be
-	// done during the relayer loop.
-	//
-	// Ref: https://github.com/umee-network/peggo/issues/55
-	return loops.RunLoop(ctx, p.logger, p.loopsDuration, func() error {
+	// Run every approximately 60 Cosmos blocks (around 5m) to allow time to receive new transactions.
+	// Running this faster will cause a lot of small batches and lots of messages going around the network.
+	// We need to remember that this call is going to be made by all the validators.
+	// This loop is configurable so it can be adjusted for E2E tests.
+	loopDuration := time.Duration(
```

Comment:
Why not pass this variable into the `peggyOrchestrator` constructor like you do for `relayerLoopDuration`?

Reply:
I had a reason for this, but now I consider it invalid lol. Fixed!